projects/kenneth_riveiro.md

```diff
@@ -1,45 +0,0 @@
-# ReactPro Tickets
```

Comment:
Please don't remove this file

Reply:
Buenas tardes ya realize el commit con el archivo que habia eliminado saludos